### PR TITLE
Update envoy-wasm sha

### PR DESCRIPTION
**Pull request by lizan**
Signed-off-by: Lizan Zhou <lizan@tetrate.io>

```release-note
None
```

### CODE REVIEW

**Comment by lizan** [comment]
@jplevyak PTAL this is mostly fixing changes by envoyproxy/envoy-wasm#274

**Comment by lizan** [comment]
@PiotrSikora ASAN is RBE timeout, what's the RBE cluster setting we're using?

**Comment by PiotrSikora** [comment]
I don't think there are any settings on the RBE side other than number of instances and their type?

**Comment by lizan** [comment]
@PiotrSikora I'm talking about the machine type, envoy-ci is on n2-standard-2 FYI

File: .bazelrc [hunk 1]
```diff
@@ -8,7 +8,7 @@ import %workspace%/envoy.bazelrc
 
 # Overrides workspace_status_command
 build --workspace_status_command=tools/bazel_get_workspace_status
-
+build:remote --remote_timeout=7200
```
Comment (yxue) [inline .bazelrc]: What's this for?

Comment (lizan) [reply .bazelrc]: Without this RBE will timeout for wee8 build, because istio-testing is using different type of machine than upstream.

**Pull request merged**